.changeset/small-kangaroos-beg.md

```diff
@@ -0,0 +1,5 @@
+---
+"jscodeshift": patch
+---
+
+fix(readme): passing `recast` options
```

Comment:
We don't need changesets for the readme, since it doesn't affect the released code. You can remove this and just keep the readme changes :)

Reply:
The changeset bot confused me. Maybe you tweak it a bit:

![image](https://github.com/user-attachments/assets/594c67f1-a81c-410a-aa5a-406be483dfeb)